views/doc.list.html.twig

```diff
@@ -3,36 +3,61 @@
 {% import _self as macros %}
 
 {% block content %}
-    <div id="searchbar" class="clearfix">
-        <input id="docsearch" type="text" placeholder="Search documentation">
-    </div>
-
-    <h1>Book</h1>
-    <ul>
-    {% for filename, data in book %}
-        {{ macros.renderFile(filename, data) }}
-    {% endfor %}
-    </ul>
-    <h1>Articles</h1>
-    <ul>
-    {% for filename, data in articles %}
-        {{ macros.renderFile(filename, data) }}
-    {% endfor %}
-    </ul>
-    <h1>FAQs</h1>
-    <ul>
-    {% for filename, data in faqs %}
-        {{ macros.renderFile(filename, data) }}
-    {% endfor %}
-    </ul>
-
-    <h1>API Docs</h1>
-    <p>If you need to work with composer as a library you may also want to check out the <a href="/apidoc/master/index.html">API Docs</a></p>
+    <article role="article">
+
+        <div id="searchbar" class="clearfix">
+            <input id="docsearch" type="search" placeholder="Type here to search documentation..." role="searchbox"
```

Comment:
do we need the explicit role here ?

Reply:
I'm not sure input type=search has default searchbox role. Got some info on that, @stof ?

Reply:
according to the spec, it should: https://www.w3.org/TR/html-aria/ (scroll until the `input type=search` in the table).
Not sure whether browsers implement it.

Reply:
Ok, will remove.